xonsh/built_ins.py

```diff
@@ -555,6 +564,16 @@ def run_subproc(cmds, captured=True):
                 except PermissionError:
                     e = 'xonsh: subprocess mode: permission denied: {0}'
                     raise XonshError(e.format(cmd[0]))
+        _stdin_file = None
+        if captured == 'object' and stdin is not None:
+            _stdin_file = tempfile.NamedTemporaryFile()
+            cproc = Popen(['cat'],
+                          stdin=stdin,
+                          stdout=PIPE)
+            tproc = Popen(['tee', _stdin_file.name],
```

Comment:
That may be.  I'm not sure.  I guess I was imagining that most Windows folks would have msysgit or similar installed, which would presumably come with these programs.  But maybe that is not a safe assumption.  We could wrap this with a check of `ON_POSIX`, and just give up on capturing stdin (which may not have a whole lot of utility).  Or we could try to make a pure-Python version.  Any preference?